CHANGELOG.md

```diff
@@ -2,7 +2,83 @@
 
 ## Unreleased
 
-  - …
+  - General data model
+    - `DataPayload` no longer needs a lifetime (#1297, #1279)
+    - Re-write ResourceKey (#1511)
+    - Rewrite ErasedDataProvider as AnyProvider (#1495)
+    - Add EitherProvider and rename IterableDataProviderCore to IterableProvider (#1455)
+    - Change DataRequest to be borrowed in BufferProvider (#1416)
+    - Replace SerdeDeDataProvider with BufferProvider (#1369, #1384)
+  - Components
+    - `icu_calendar`:
+      - Julian, Japanese, and Buddhist calendars (#1351, #1394, #1305)
+      - `DateTimeFormat` integration (#1339)
+      - Bugfix around arithmetic (#1352)
+    - `icu_datetime`:
+      - Week-of-year support (#1206)
+      - `DateTimeFormat::resolve_components()` (#1362)
+      - Era formatting (#1346)
+      - `TimeZoneFormatConfig` (#1256)
```

Comment:
There is no issue for this (only a PR), but Sam fixed in issue in which some localized strings for the `"standard"` time-zone variant were previously missing from our packaged data.

I think this is worth mentioning as a bug fix.